website/static/js/comment.js

```diff
@@ -273,6 +273,11 @@ var CommentModel = function(data, $parent, $root) {
     self.isAbuse = ko.observable(data.attributes.is_abuse);
     self.canEdit = ko.observable(data.attributes.can_edit);
     self.hasChildren = ko.observable(data.attributes.has_children);
+    self.hasReport = ko.observable(data.attributes.has_report);
+    self.isDeletedAbuse = ko.observable(self.isDeleted() && self.isAbuse());
+    self.isDeletedNotAbuse = ko.observable(self.isDeleted() && !self.isAbuse());
+    self.isAbuseNotDeleted = ko.observable(!self.isDeleted() && self.isAbuse());
```

Comment:
These 'calculated' ones should probably be `ko.pureComputed`, not `.observable`. Could explain some UI bugs I was seeing